### DISCUSSION TITLE
How to customize model validation error response when there are more than 1 error

### DISCUSSION THREAD

**Discussion by ghost** [laravel/framework#50232] [General]
Hi, i wanna customize the section of message which last parenthesis part when error happens.Which translation key i should override?

i see source code of laravel validations there is no such thing

```json

{ "message": "name tələb olunur. (and 2 more errors)", <-- here
    "errors": {
        "name": [
            "name tələb olunur."
        ],
        "email": [
            "email tələb olunur."
        ],
        "password": [
            "password tələb olunur."
        ]
    }
```

**Comment by henzeb** [comment]
simply put, you can't. There are several PR's that tried to alter this behavior, but Taylor doesn't want this to change. This is not for convenience of the frontend developer.

**Reply by ghost** [reply]
Actually after this i deep dive source code and see that this is hardocded feature in ValidationException class but there is a solution which people tries to catch ValidationException using middleware then changes message but still horrible solution and i do not understand why this is not implemented.
